test-it/ExpectedResults/UnflattenFromJSONEmptyArray.vtr

```diff
@@ -0,0 +1,8 @@
+1: () (false 0 '')
+2: (()) (false 0 '')
```

Comment:
I think that's an excellent idea.  Using parens for both clusters and arrays has always bothered me.

Reply:
I would only say that proper JSON formatting is much more verbose. ie the key names are duplicated in an object whereas the via formatting omits them.

`(false 0 '')` => `{"status": false, "code": 0, "source":""}`

And for each element instance in an array that becomes super verbose:

`((false 0 '') (false 0 '') (false 0 ''))` =>
`[{"status": false, "code": 0, "source":""}, {"status": false, "code": 0, "source":""}, {"status": false, "code": 0, "source":""}]`

I would probably vote for keeping it as is / vote for arrays just being square brackets:
`[(false 0 '') (false 0 '') (false 0 '')]`

But I don't feel super strongly about it.

Reply:
Although I may have misread, @sanmut were you just advocating for changing () to [] for arrays not representing everything as valid JSON? If so, sure, I can't think of a reason why not.

Reply:
Yeah, actually, I didn't read his comment that way, but if he meant using full JSON formatting with field names, I'd be against that because it's so verbose.
I think using [] for arrays is absolutely a good idea though.

Reply:
I meant only replacing parentheses () to brackets [] when I wrote this. I see your points about field names in JSON format being verbose to format and print clusters.

I am afraid if we choose to format arrays with square brackets, we should probably fix default values in constants/variables also to use square brackets !!?

I was wondering 
1. if there was any reason to choose parentheses for both arrays and clusters in formatting and parsing default values.
2. Would this break something else anyone aware of in via/Vireo.